PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD003.xml

```diff
@@ -0,0 +1,231 @@
+                                    <ab type="ruling">The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
+                                    <locus from="1r" to="62r"/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>The manuscript can be dated to the fourteenth or fifteenth century based on palaeographic features. For example, the letter መ shows a tendency for loop separations, ሎ demonstrates the
+                                    vowel marker with a linking line, ጵ exhibits the form used before the sixteenth century. Additionally, the word እግዚአብሔር is 
+                                    written as a single word. Overall, the manuscript displays the standard orthography 
+                                    of medieval Ethiopic. </desc>
+                               <seg type="ink">Black, red.</seg>
+                                <date notBefore="1400" notAfter="1500"> </date>
+                               </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                               <item xml:id="e1">
+                                   <locus target="#44v #45"></locus>
+                                   <desc>The name <persName ref="PRS14538AskalaGiyorgis"/> is written in supplications over erasure, 
+                                       possibly close to the time of the production of the main text. </desc>
+                                   <q xml:lang="gez"><seg part="I">ወናዝዝ፡ ሐዘንየ፡ ሊተ፡ ለገብርከ፡<persName ref="PRS14538AskalaGiyorgis">
+                                       አስካለ፡ ጊዮርጊስ፡</persName>
+                                      </seg></q>
+                                </item>
+                               </list>
+                        </additions>                     
+                           
+                        <bindingDesc>
+                            <binding contemporary="unknown" xml:id="binding">
+                                <decoNote xml:id="b1">The boards of the manuscript are missing. The textblock is held together by the sewing.</decoNote> 
+                             <decoNote xml:id="b2" type="SewingStations">4</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    
+                    <history>
+                        <origin>
+                          <origDate notBefore="1400" notAfter="1500" evidence="lettering">
+                                1400–1500 </origDate>.
+                        </origin>
+                        <provenance>No colophon notes are present in the manuscript, but the owner,
```

Comment:
the **current** owner

Reply:
If the current owner is `PRS14417MarigetaDamseEndalew `how can he have the role of the `scribe `as well?
At least one of the previous owners was clearly `PRS14538AskalaGiyorgis`, but there was at least one before